src/main/java/me/qoomon/maven/gitversioning/GitVersioningModelProcessor.java

```diff
@@ -290,13 +292,40 @@ private Model processModel(Model projectModel) throws IOException {
                         sessionProjectDirectories.add(new File(projectModel.getProjectDirectory(), module).getCanonicalPath());
                     }
                 }
+
+                for (Model previousModel : virtualProjectModelCache.values()) {
```

Comment:
May you explain why you only update dependency version if `isProjectPom`? My guess was to put it in line 266.

Reply:
You are right. We need to update them as well.

Reply:
@qoomon Why do we need to update other poms too? I think this is only relevant to project's pom files. I think it is correct place now?

Reply:
@cemo non relevant pom files are already skipped by line 189. The purpose of this code block is just to add mojo plugin.